lib/sms.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var Nexmo = require('nexmo')
+var P = require('bluebird')
+
+module.exports = function (log, translator, templates, config) {
```

Comment:
nit: `config` -> `smsConfig`, so it is clear that it is not the full server config. nit though!